BrowserKit/Sources/ComponentLibrary/Buttons/LinkButtonViewModel.swift

```diff
@@ -14,21 +15,21 @@ public struct LinkButtonViewModel {
 
     public let title: String
     public let a11yIdentifier: String
-    public let fontSize: CGFloat
+    public let font: UIFont?
     public let contentInsets: NSDirectionalEdgeInsets
     public let contentHorizontalAlignment: UIControl.ContentHorizontalAlignment
 
     public init(title: String,
                 a11yIdentifier: String,
-                fontSize: CGFloat = 16,
+                font: UIFont? = FXFontStyles.Regular.callout.scaledFont(),
```

Comment:
nit: remove optional?

Reply:
I just removed the optional and pushed the modifications